app/src/pages/header/movieDetails/MovieDetails.jsx

```diff
@@ -1,33 +1,32 @@
 import PropTypes from 'prop-types';
 import { useState, useEffect } from 'react';
-import moviesFromJson from '@assets/data/movies';
 import { MovieDetailsImg } from '@pages/header/movieDetails/components/MovieDetailsImg';
 import { MovieDetailsComponent } from '@pages/header/movieDetails/components/layouts/MovieDetailsComponent';
 import { MovieDetailsName } from '@pages/header/movieDetails/components/MovieDetailsName';
 import { MovieDetailsText } from '@pages/header/movieDetails/components/MovieDetailsText';
 import { MovieDetailsAccentText } from '@pages/header/movieDetails/components/MovieDetailsAccentText';
 import { AgeTimeContainer } from '@pages/header/movieDetails/components/layouts/AgeTimeContainer';
+import { useSelector } from 'react-redux';
 
 function MovieDetails(props) {
+    const movies = useSelector(state => state.movies);

```

Comment:
useSelector(moviesSelector);

Reply:
const state = getState()

const movies = moviesSelector(state)

Reply:
Done: added selectors, used in all `useSelector` hooks